### DISCUSSION TITLE
Advance tuning support for CPU model and topology

### DISCUSSION THREAD

**Discussion by gqlo** [saltstack/salt#67287] [Feature Requests]
Making [CPU model and topology](https://libvirt.org/formatdomain.html#elementsCPU) configurable enhances resource utilisation in big machines.

**Comment by s0undt3ch** [comment]
Thank You for additionally providing the PR implementing the feature.

**Comment by gqlo** [comment]
> Thank You for additionally providing the PR implementing the feature.

My pleasure. :)
